values.yaml

```diff
@@ -25,7 +25,8 @@ ingress:
     # Fill the secretName if you want to use the certificate of 
     # yourself when Harbor serves with HTTPS. A certificate will 
     # be generated automatically by the chart if leave it empty
-    secretName: ""
+    secretName: &tls_secretName ""
+    notarySecretName: *tls_secretName
```

Comment:
@jw-s Could you add some comment here to explain how to use these two config items?

Reply:
Done